### PR TITLE
Updated libraries

### PR DESCRIPTION
**Pull request by mishrabhilash**
- kotlinVersion 1.3.11  ->  1.3.21
 - navVersion 1.0.0-alpha08  ->  1.0.0

### CODE REVIEW

**Comment by Aexyn** [comment]
Latest Kotlin -> 1.3.30

**Comment by MatushkaRossiya** [comment]
I think that Navigation library should be actually updated to version 2.0.0, as MvRx project is now based on AndroidX dependencies.

**Comment by mishrabhilash** [comment]
@Aexyn and @MatushkaRossiya, I have updated to commented versions.

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlinVersion = '1.3.11'
+    ext.kotlinVersion = '1.3.21'
```
Comment (gpeal) [inline build.gradle]: 1.3.31 is the latest now

Comment (gpeal) [reply build.gradle]: @mishrabhilash

Comment (mishrabhilash) [reply build.gradle]: upgraded this and some other libraries

**Pull request merged**